sdks/python/apache_beam/io/components/adaptive_throttler.py

```diff
@@ -121,7 +121,26 @@ def successful_request(self, now):
 
 class ReactiveThrottler(AdaptiveThrottler):
   """ A wrapper around the AdaptiveThrottler that also handles logging and
-  signaling throttling to the SDK harness.
+  signaling throttling to the SDK harness using the provided namespace.
+
+  For usage, instantiate one instance of a ReactiveThrottler class for a PTransform.
+  When making remote calls to a service, preface that call with the throttle()
+  method to potentially pre-emptively throttle the request. After that call,
```

Comment:
```suggestion
  method to potentially pre-emptively throttle the request. This will throttle future
  calls based on the failure rate of preceding calls, with higher failure rates leading
  to longer periods of throttling to allow system recovery.
```

Feel free to modify the wording, but I think we need something like this to explain what users can expect